upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -205,55 +143,50 @@ There you can find links to upgrade notes for other versions too.
-    - copy necessary configuration [shopsys_frontend_api.yaml from Github](https://github.com/shopsys/shopsys/blob/master/project-base/config/packages/shopsys_frontend_api.yaml) to `config/packages/shopsys_frontend_api.yaml`
+    - run these steps only in case you have not recently updated to Symfony Flex as is described at beginning of this file
+        - add `shopsys/frontend-api` dependency with `composer require shopsys/frontend-api`
+        - register necessary bundles in `config/bundles.php`
+            ```diff
+                Shopsys\FormTypesBundle\ShopsysFormTypesBundle::class => ['all' => true],
+            +   Shopsys\FrontendApiBundle\ShopsysFrontendApiBundle::class => ['all' => true],
+            +   Overblog\GraphQLBundle\OverblogGraphQLBundle::class => ['all' => true],
+            +   Overblog\GraphiQLBundle\OverblogGraphiQLBundle::class => ['dev' => true],
+                Shopsys\GoogleCloudBundle\ShopsysGoogleCloudBundle::class => ['all' => true],
+            ```
+        - add new route file [`config/routes/frontend-api.yaml`](https://github.com/shopsys/shopsys/blob/master/project-base/config/routes/frontend-api.yaml) from GitHub
+        - add new route file [`config/routes/dev/frontend-api-graphiql.yaml`](https://github.com/shopsys/shopsys/blob/master/project-base/config/routes/dev/frontend-api-graphiql.yaml) from GitHub
+        - copy [type definitions from Github](https://github.com/shopsys/shopsys/tree/master/project-base/config/graphql/types) into `config/graphql/types/` folder
+        - copy necessary configuration [shopsys_frontend_api.yaml from Github](https://github.com/shopsys/shopsys/blob/master/project-base/config/packages/shopsys_frontend_api.yaml) to `config/packages/shopsys_frontend_api.yaml`
+        - update your `security.yaml` configuration [using this diff](https://github.com/shopsys/project-base/commit/3e9a056f032b3fb49e4aaac912fa89cae13725c6#diff-e092a3a494858e808395bb5a24bb8f83)
     - copy [tests for FrontendApiBundle from Github](https://github.com/shopsys/shopsys/tree/master/project-base/tests/FrontendApiBundle) to your `tests` folder
+    - update your `easy-coding-standard.yaml` file:
+        - add these in `ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff` part
+            ```diff
+                - '*/tests/FrontendApiBundle/Functional/Image/ProductImagesTest.php'
+                - '*/tests/FrontendApiBundle/Functional/Payment/PaymentsTest.php'
+                - '*/tests/FrontendApiBundle/Functional/Transport/TransportsTest.php'
+                - '*/tests/FrontendApiBundle/Functional/Order/MultipleProductsInOrderTest.php'
+            ```
+    - update your `build.xml` [using this diff](https://github.com/shopsys/project-base/commit/02ca46eb77d0c96dc6ff1903f434ebb0537248bd#diff-2cccd7bf48b7a9cc113ff564acd802a8)
     - enable Frontend API for all domains by `./phing frontend-api-enable` command (you can manage domains in `config/packages/frontend_api.yaml`)
```

Comment:
Is there any info about the need to configure `build.xml` and allow FE API phing commands ?

Reply:
Yes right above this line :)